crates/utils/src/settings/structs.rs

```diff
@@ -55,28 +55,62 @@ pub struct PictrsConfig {
 }
 
 #[derive(Debug, Deserialize, Serialize, Clone, SmartDefault, Document)]
-#[serde(default, deny_unknown_fields)]
```

Comment:
Can't use `deny_unknown_fields` in combination with `flatten`: https://serde.rs/container-attrs.html#deny_unknown_fields